libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -1437,10 +1437,9 @@ AP_InertialSensor::_init_gyro()
 
     // we've kept the user waiting long enough - use the best pair we
     // found so far
-    hal.console->printf("\n");
```

Comment:
This line matches the emitting of "*" above; shouldn't change one without the other.

Do these messages actually make it to the GCS?